pkg/trace/api/api.go

```diff
@@ -72,7 +71,6 @@ type HTTPReceiver struct {
 
 	maxRequestBodyLength int64
 	debug                bool
-	refuse               int64 // when set to 1 agent will refuse payloads
```

Comment:
Ok, good we got rid of that `refuse` flag, at least everything is now "rate based".